javascript/javascript2/week3/The-fastest-presser-in-this-realm.js

```diff
@@ -0,0 +1,81 @@
+"use strict"
+
+const userInput = document.getElementById('number');
+const button = document.getElementById('button');
+const gameInfo = document.getElementById('game_info');
+const restart = document.getElementById('restart_button');
+
+function gameEnd() {
+  document.removeEventListener("keypress", keyPresses);
+  if (countOne > countTwo) {
+    gameInfo.innerHTML = "WINNER : S-key user";
+    var confettiSettings = {
+      target: 'my-canvas-one'
+    };
+    var confetti = new ConfettiGenerator(confettiSettings);
+    confetti.render();
+  } else if (countOne < countTwo) {
+    gameInfo.innerHTML = "WINNER: L-key user ";
+    var confettiSettings = {
+      target: 'my-canvas-two'
+    };
+    var confetti = new ConfettiGenerator(confettiSettings);
+    confetti.render();
+  } else if (countS === 0 && countL === 0) {
+    gameInfo.innerHTML = "No key presses!Try again";
+  } else if (countS === countL) {
+    gameInfo.innerHTML = "Its a Tie ";
+    console.log('Its a Tie');
+
+  }
+}
+
+button.addEventListener('click', function gameStart() {
+  let time = userInput.value;
+  //  console.log(Number(time));
+  if (time === "" || Number.isNaN(Number(time))) {
+    gameInfo.innerHTML = "Enter time in seconds"
+  } else {
+    gameInfo.innerHTML = "Game will start and Time limit is set for " + time + " seconds";
+
```

Comment:
Once countdown started no need to show the message "Game will start and time limit is set for x seconds". 
Use `HTMLElement.hidden` property at appropriate place.